### PR TITLE
poc parametric test integration

### PR DESCRIPTION
**Pull request by robertomonteromiguel**
## Description

Integrate parametric tests in system-tests as new scenario.

## Additional notes

I copy-pasted code from system-tests/parametric:

* Utilities (utils, protos and spec) from system-tests/parametric to system-tests/tests/parametric
* The POC only contains a unit tests file. Copy-paste _system-tests/parametric/test_tracer.py_ to _system-tests/tests/parametric/test_tracer.py_. Decorate it with @scenario
* Copy from _system-tests/parametric/apps_ to system-tests/utils/builds/docker/<lang>/parametric

### CODE REVIEW

**Comment by cbeauchesne** [comment]
parametric-poc CI fails, do you need a review antway ?

**Comment by robertomonteromiguel** [comment]
> parametric-poc CI fails, do you need a review antway ?

Yes, it fails PHP, but it also fails in the current parametric tests.

File: utils/_context/_scenarios.py [hunk 1]
```diff
@@ -653,6 +653,16 @@ def _extra_weblog_warmup(self):
...
 
 
+class ParametricScenario(_Scenario):
+    @property
+    def host_log_folder(self):
+        return "logs_parametric"
+
+    @property
+    def library(self):
+        return LibraryVersion(os.getenv("TEST_LIBRARY", "java"), "0.00")
```
Comment (cbeauchesne) [inline utils/_context/_scenarios.py:663]:
```suggestion
        return LibraryVersion(os.getenv("TEST_LIBRARY", "**not-set**"), "0.00")
```

File: utils/_context/_scenarios.py [hunk 2]
```diff
@@ -653,6 +653,16 @@ def _extra_weblog_warmup(self):
         time.sleep(WARMUP_LAST_SLEEP_DURATION)
 
 
+class ParametricScenario(_Scenario):
+    @property
+    def host_log_folder(self):
+        return "logs_parametric"
+
```
Comment (cbeauchesne) [inline utils/_context/_scenarios.py:660]:
```suggestion
def configure(self):
    super().configure()
    assert "TEST_LIBRARY"  in os.env
```

**Pull request merged**